src/main/java/uk/ac/ebi/eva/pipeline/configuration/BeanNames.java

```diff
@@ -62,9 +62,11 @@ public class BeanNames {
     public static final String ACCESSION_IMPORT_STEP = "accession-import-step";
 
     public static final String AGGREGATED_VCF_JOB = "aggregated-vcf-job";
+    public static final String AGGREGATED_VARIANT_LOAD_VCF_JOB = "aggregated-variant-load-vcf-job";
```

Comment:
Personally I'd prefer naming like this throughout:
```suggestion
    public static final String AGGREGATED_VCF_LOAD_JOB = "aggregated-vcf-load-job";
```
since I think of it as a "load job" being applied to an "aggregated (or genotyped) vcf".